libkrun/libkruntime.c

```diff
@@ -1,5 +1,10 @@
 /*
- * C support functions.
+ * C support functions for timing.
+ *
+ * Although this is a library, any error will exit indicating failure. Krun
```

Comment:
I'm fine with that -- it's just the English isn't very easy to parse at the moment.